CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## 1.0.1
```

Comment:
thank you for adding a change log! But this will be 1.1.0 since this release will introduce new features.